scap-security-guide.spec.in

```diff
@@ -77,31 +85,31 @@ rm %{buildroot}%{_datadir}/xml/scap/ssg/content/*-cpe-dictionary.xml
 # We do this after the filtering on Fedora because we don't ship JBossEAP5 datastreams
 cp -a JBossEAP5/eap5-* %{buildroot}%{_datadir}/xml/scap/ssg/content/
 
-# Add in oscap-scan files
-#mkdir -p %{buildroot}%{_sysconfdir}/sysconfig/
-#cp -a {*,*/*}/input/oscap-scan-* %{buildroot}%{_sysconfdir}/sysconfig/
+# Docs
+mkdir -p %{buildroot}/%{_docdir}/%{name}/guides
+cp -a RHEL/6/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+cp -a RHEL/7/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+cp -a Firefox/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+cp -a JRE/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+# outside of the normal build system, different guide
+cp -a JBossEAP5/docs/JBossEAP5_Guide.html %{buildroot}/%{_docdir}/%{name}/guides
+
+%if 0%{?fedora}
+cp -a Fedora/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+cp -a Chromium/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+#cp -a Webmin/output/*-guide-*.html %{buildroot}/%{_defaultdocdir}/%{name}/guides
+%endif
 
 %files
 %{_datadir}/xml/scap
 %{_datadir}/%{name}/remediation_functions
+%{_datadir}/%{name}
```

Comment:
The previous 2 lines cause:
```
warning: File listed twice: /usr/share/scap-security-guide/remediation_functions
```

Drop the `%{_datadir}/%{name}/remediation_functions` line. This is what the Fedora spec does.